aligner/corpus.py

```diff
@@ -67,9 +67,15 @@ def __init__(self, dirname, opts):
                 logging.error("Phone '{}': not /{}/.".format(phone,
                                                       VALID_PHONE))
                 exit(1)
-        # dictionary
-        self.dictionary = opts["dictionary"]
-        self.thedict = PronDict(self.dictionary, self.phoneset)
+        # dictionaries
+        self.dictionaries = []
+        if "dictionary" in opts and opts["dictionary"]:
```

Comment:
I would rather insist that `opts[dictionary]` is always a list and modify existing YAML files.